mdns.go

```diff
@@ -36,7 +36,10 @@ type MDNS struct {
 func (m MDNS) ReplaceLocal(input string) string {
 	// Replace .local domain with our configured custom domain
```

Comment:
wouldn't this doc string be outdated by the change?

Reply:
Ah, yes. Good catch.

Reply:
In fact, that whole function name is now outdated. Fixed in the latest commit.